src/calculateRentalCost.js

```diff
@@ -3,8 +3,23 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const rentPrice = 40;
+  const shortTerm = 3;
+  const shortTermDiscount = 20;
+  const longTerm = 7;
+  const longTermDiscount = 50;
```

Comment:
https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md

we usually name constants in uppercase